docs/man/sat-bootsys.8.rst

```diff
@@ -159,6 +159,9 @@ These options apply to both the ``shutdown`` and ``boot`` actions.
 **--bos-version BOS_VERSION**
         The version of the BOS API to use when launching BOS sessions.
 
+**--cfs-version CFS_VERSION**
+        The version of the CFS API to use when launching CFS sessions.
```

Comment:
We only query the CFS API for active sessions. 

```suggestion
        The version of the CFS API to use when querying CFS sessions.
```

We don't launch any CFS sessions in `sat bootsys` like we do for BOS sessions in the `bos-operations` stage. Perhaps the documentation for `--bos-version` above would more accurately be "The version of the BOS API to use when launching or querying BOS sessions."

Reply:
Addressed